### PR TITLE
Bugs fix

### PR DESCRIPTION
**Pull request by s0nicboOm**
Explain what this PR does.

### CODE REVIEW

File: numaprom/clients/prometheus.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def query_metric(
             df = pd.concat(frames, ignore_index=True)
             df.sort_values(by=["timestamp"], inplace=True)
             df["timestamp"] = pd.to_datetime(df["timestamp"], unit="s")
-            df.reset_index(drop=True, inplace=True)
+
```
Comment (ab93) [inline numaprom/clients/prometheus.py:55]: Can you raise a custom exception, e.g. `InvalidDatashapeError`, if the data is not of the correct shape?

**Review by ab93 — COMMENTED** [review]
Small comment

File: numaprom/clients/prometheus.py (hunk 1 above)
Comment (s0nicboOm) [reply numaprom/clients/prometheus.py:55]: Added it in training

File: numaprom/udsink/train_rollout.py [hunk 2]
```diff
@@ -147,6 +155,9 @@ def train_rollout(datums: Iterator[Datum]) -> Responses:
             responses.append(Response.as_success(_datum.id))
             continue
 
+        if train_df.shape[1] != get_model_config(metric_config).n_features:
+            raise InvalidDataShapeError(f"Train data shape error. Input shape: {train_df.shape}")
```
Comment (ab93) [inline numaprom/udsink/train_rollout.py:159]: Can you add, what is the expected shape?

**Review by ab93 — APPROVED** [review]
added another comment. But otherwise LGTM

**Pull request merged**